docs/quickstart/crud-operations.rst

```diff
@@ -8,8 +8,8 @@
 
 .. _crud-operations:
 
-Create, Display, Search Records
-===============================
+Create, Display, Upload File, Download File, Search Records
```

Comment:
Now the header is just repeating the sub-headers which seems unnecessary. What do you think?